### DISCUSSION TITLE
SLEEF is changing maintainer

### DISCUSSION THREAD

**Discussion by blapie** [shibatch/sleef#472] [General]
Dear SLEEF users and contributors,

We are in the process of changing global maintainers for SLEEF. Pending issues and PRs will be addressed in timely manner.
We will post more information on this thread once we are ready to take over, hopefully in the upcoming weeks.

Apologies for any further delay in responses to your issues, PRs, ...

**Comment by darshanp4** [comment]
any update on this!

**Comment by blapie** [comment]
Dear users and contributors,
 
As you may have noticed, SLEEF maintenance is now assured after a period of relative silence. A small team of engineers at Arm Manchester has taken over the role of global maintainers for SLEEF. Although we may have limited bandwidth for more in depth contributions, we will be dedicated to
- keep hold of the backlog: reviewing, testing and merging PRs, fixing pending issues, creating new ones, ...
- manage releases, github page, and documentation,
- and keep discussions going on github.
 
All of this will be done in the best interest of SLEEF users, cross-platform performance, portability and future maintenance.
